reconcile/gitlab_labeler.py

```diff
@@ -10,6 +11,60 @@
 QONTRACT_INTEGRATION = 'gitlab-labeler'
 
 
+def get_app_list() -> Dict:
+    """
+    Get a list of all services managed by app-interface with the parent app
+    and the onboarding status of the service.
+    """
+    apps = queries.get_apps()
+    app_info = {a['name']: {'onboardingStatus': a['onboardingStatus'],
+                'parentApp': a['parentApp']} for a in apps}
+    return app_info
+
+
+def get_parents_list() -> List[str]:
+    """
+    Get a set of all service names that are parents of another service
+    """
+    parent_set = set()
+    apps = queries.get_apps()
+    for a in apps:
+        if a['parentApp'] is not None:
+            parent_set.add(a['parentApp']['name'])
+
+    return list(parent_set)
+
+
+def guess_onboarding_status(changed_paths: Iterable[str],
+                            apps: Dict[str, Dict], parent_apps:
+                            Set[str]) -> Optional[str]:
+    """
+    Guess the service name of a given MR from the changed paths of the
+    MR. This will allow to add the onboarding status to the MR's as label
+    """
+    labels = set()
+    for path in changed_paths:
+        if 'data/services/' in path:
+            path_slices = path.split('/')
```

Comment:
i recommend to assign the different slices to variables with meaningful names instead of using `path_slices[2]` (for example)